ocs_ci/ocs/resources/pod.py

```diff
@@ -708,13 +710,14 @@ def get_all_pods(
     return pod_objs
 
 
-def get_ceph_tools_pod(skip_creating_pod=False, namespace=None):
+def get_ceph_tools_pod(skip_creating_pod=False, wait=False, namespace=None):
```

Comment:
with the new arg, there is a code changes at many places. This wait arg seems useful at this point only to stretch cluster negative test cases, should we consider creating a separate wrapper function for this alone? maybe simply keep checking  for the tool box pod using timeout sampler till the give time until it reaches running state

Reply:
This doesn't affect the already existing use case because, wait = False by default which is equivalent to not using it. and when wait=False it acts just like the code previous to this change. so it doesn't matter. Also this can be used in places where we may need to wait tool box pod, like some other DR scenarios.